src/Classifier.php

```diff
@@ -79,6 +79,10 @@ public function find(): iterable
         $this->scanFiles();
 
         $classesToFind = get_declared_classes();
+        $directories = array_map(
+            static fn($directory) => DIRECTORY_SEPARATOR === '\\' ? str_replace('/', '\\', $directory) : $directory,
```

Comment:
Please check `DIRECTORY_SEPARATOR === '\\'` outside the function and use `str_replace` with array arguments:
```php

$directories = $this->directories;

$isWindows = DIRECTORY_SEPARATOR === '\\';
if ($isWindows) {
    $directories = str_replace('/', '\\', $this->directories);
}
```

Reply:
@rustamwin `str_replace` support batch replacing, no need to iterate and replace individually